misk-aws/src/main/kotlin/misk/dynamodb/DyTimestampedEntity.kt

```diff
@@ -0,0 +1,19 @@
+package misk.dynamodb
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGenerateStrategy
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedTimestamp
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable
+import java.util.Date
+
+@DynamoDBTable(tableName = "")
+abstract class DyTimestampedEntity : DyEntity() {
```

Comment:
Do we have a use case for a non-timestamped entity?

Reply:
I do not. But I know there are use cases of non-timestamped entities in SQL, so I took the same approach here